scripts/variantstore/beta_docs/run-your-own-samples.md

```diff
@@ -0,0 +1,176 @@
+
+---
+
+**Warning:**        
+The workflow in the GVS beta workspace is configured based on the format of the TSV file. To avoid reconfiguring the workflow, do **not** rearrange or rename the columns in the TSV file.
+
+---
+
+7. Follow the steps in [How to use the Data Uploader](https://support.terra.bio/hc/en-us/articles/4419428208411) to **upload your data and TSV file** to Terra.
+
+## Run the workflow
+
+Now that your samples are loaded into data table in Terra, it’s time to setup and run the GVS workflow! The workflow is configured to call inputs from and write outputs to the sample_set table.
+
+1. **Select the workflow** from the Workflows tab.
+1. In the configuration page, select the **sample_set entity** in Step 1.
+1. Create a new sample_set dataset in Step 2.
+    1. Select **Create a new sample_set from selected samples**.
+    1. Click on the **arrow next to the tickbox** in the header row of the table.
+    1. Select All in the dropdown menu that appears to select all of the samples.
+    1. If you do **not** want to include the example data in your callset, **click on the tickbox** next to each of the 10 samples to deselect them.
+    1. At the bottom of the page, **name your new sample_set**. This will appear in the sample_set data table.
+    1. Click **OK** to create the sample_set.
+1. Configure the workflow inputs.
+    1. Enter a **name for the callset** as a string with the format “*CALLSET_NAME*” for the `callset_identifier` variable. This string is used as to name several variables and files and should begin with a letter. Valid characters include A-z, 0-9, “.”, “,”, “-“, and “_”.
+    1. Enter the name of your **BigQuery dataset** as a string with the format “*DATASET_NAME*” for the `dataset_name` variable.
+    1. Enter the name of the **GCP project** that holds the BigQuery dataset as a string with the format “*PROJECT_NAME*” for the `project_id` variable.
+1. **Save** the workflow configuration.
+1. **Run** the workflow.
+
+### Adding additional data to the BigQuery dataset
+The workflow imports GVCFs to the BigQuery dataset. The example GVCFs for the workspace are listed as a set in the workspace sample_set table. Multiple sample sets may be added to the same BigQuery dataset to appear in the same callset, but the workflow only runs on one sample set at a time. 
+
+If you run the workflow with the example data pre-loaded into the workspace, and then load additional data into that same dataset, the example data will be in your callset. Be sure to make a new dataset for your own data if you run the example data (unless you want the example data in there too–the more the merrier!).
+
+### Important configuration notes
+
+By default, the workflow is set up to write outputs to the workspace Google bucket. If you want to write the outputs to a different cloud storage location, you can specify the cloud path in the `extract_output_gcs_dir` optional input in the workflow configuration. 
+
```

Comment:
<workspace bucket id>/<submission id>/GvsJointVariantCalling/<workflow id>/call-GvsUnified/GvsUnified/<task id, note that this is the only option in this dir and doesn't need to be looked up>/call-GvsExtractCallset/GvsExtractCallset/<task id, note that this is the only option in this dir and doesn't need to be looked up>/call-SplitIntervals/glob-<glob id>/